packages/runtime/src/next/document.tsx

```diff
@@ -36,7 +38,7 @@ export class Document extends NextDocument {
     return (
       <Html>
         <Head>
-          <PreviewModeScript isPreview={isPreview} />
+          <PreviewModeScript isPreview={isPreview} appOrigin={this.props.appOrigin} />
```

Comment:
Support running Pages router apps in local builder.

Reply:
I see now what you're doing with the document's `static defaultProps`. Seems like the only choice before was to use a custom `Document` and passing it in as a prop. I guess this allows to extend `Document` without having to re-implement the `render()` method if all you want to do is add a custom app origin?

Reply:
> I guess this allows to extend `Document` without having to re-implement the `render()` method if all you want to do is add a custom app origin?

Yes.